tests/test_endpoints.py

```diff
@@ -378,6 +407,7 @@ def test_purchase_bills(self):
         self.assertEndpointReached(self.companyfile.purchase_bills.put_miscellaneous, {'uid': UID, 'data': DATA}, 'PUT', f'/{CID}/Purchase/Bill/Miscellaneous/{UID}/')
         self.assertEndpointReached(self.companyfile.purchase_bills.post_miscellaneous, {'data': DATA}, 'POST', f'/{CID}/Purchase/Bill/Miscellaneous/')
         self.assertEndpointReached(self.companyfile.purchase_bills.delete_miscellaneous, {'uid': UID}, 'DELETE', f'/{CID}/Purchase/Bill/Miscellaneous/{UID}/')
+        self.assertEmptyDeleteResponseHandled(self.companyfile.purchase_bills.delete_miscellaneous, {'uid': UID}, f'/{CID}/Purchase/Bill/Miscellaneous/{UID}/')
```

Comment:
There's no need to add this test for multiple endpoints -- we're mocking the endpoints anyway.

Reply:
My first approach was to tack something onto `assertEndpointReached` to check that it handles empty strings from  `DELETE` requests. What do you suggest?

Reply:
I think the `assertEndpointReached ` is chiefly interested in testing that we generate out the endpoint urls correctly.

So I'd be testing the return value separately.